### PR TITLE
Add bot cheatsheet

### PR DESCRIPTION
**Pull request by maelle**
TODOs

- [x] Get review for English version.
- [x] Create translation.
- [x] Get review for Spanish version.

### CODE REVIEW

**Comment by mpadge** [comment]
How about putting one additional sentence on a few of them like:

> *When to use?* Generally only on pre-submission enquiries, or when authors otherwise indicate that package has substantially changed.

or

> *When to use?* Only after all reviews have been approved.

I can have a go at that if you like.

**Comment by maelle** [comment]
@mpadge yes please add such suggestions as you see fit :pray:

**Comment by maelle** [comment]
@mpadge I added some notes, could you please review?

File: bot_cheatsheet.Rmd [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+## For everyone {.unnumbered}
+
+### See the list of commands available to you {.unnumbered}
+
+If you need a quick reminder!
+
+```markdown
+@ropensci-review-bot help
+```
+
```
[the remark this answers is not in the document]
Comment (mpadge) [reply bot_cheatsheet.Rmd:12]: Actually, maybe that's better after the "code of conduct" command, and stated in a more general form?

[the remark this answers is not in the document]
Comment (maelle) [reply bot_cheatsheet.Rmd:12]: what do you mean by more general form?

who does this cleaning and have we formalized it elsewhere btw?

[the remark this answers is not in the document]
Comment (mpadge) [reply bot_cheatsheet.Rmd:12]: Nope, no formalized cleaning, and nobody in particular does it, but it definitely seems to be done, so i'd say no current need for formalisation, but still a useful thing to write there.

And by "more general", i just mean to move that comment down, and generalise it to refer to both `help` and `c-o-c`, if that makes sense?

**Review by yabellini — COMMENTED** [review]
This is great.  I made changes to remove the gender mark.  It is a very clear guide.

File: bot_cheatsheet.es.Rmd [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Comandos bot
```
Comment (yabellini) [inline bot_cheatsheet.es.Rmd:1]:
```suggestion
# Comandos del bot
```

File: bot_cheatsheet.es.Rmd [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+## Para todos {.unnumbered}
```
Comment (yabellini) [inline bot_cheatsheet.es.Rmd:3]:
```suggestion
## Para todas las personas que participan {.unnumbered}
```

**Comment by maelle** [comment]
Thank you @yabellini!!

**Pull request merged**